src/sentry/utils/cloudfunctions.py

```diff
@@ -53,11 +64,11 @@ def upload_function_files(client, code, env_variables):
     f.seek(0)
 
     logger = logging.getLogger("sentry.functions")
-    logger.info(f"The region is {SENTRY_FUNCTIONS_REGION}")
+    logger.info(f"The region is {settings.SENTRY_FUNCTIONS_REGION}")
 
     upload_url = client.generate_upload_url(
         request=GenerateUploadUrlRequest(
-            parent=f"projects/{SENTRY_FUNCTIONS_PROJECT_NAME}/locations/{SENTRY_FUNCTIONS_REGION}"
+            parent=project_location_function_name(True, True, False),
```

Comment:
@vpandiarajan20 It's hard to understand what the arguments mean in this context. It would be better to use named arguments for clarity as well as omitting any false arguments.